tests/unit/s2n_post_handshake_test.c

```diff
@@ -101,6 +105,34 @@ int main(int argc, char **argv)
 
             EXPECT_SUCCESS(s2n_connection_free(conn)); 
         }
+
+        /* Functional test: Multiple post handshake messages can be received in the same record */
+        {
+            struct s2n_connection *conn;
+            EXPECT_NOT_NULL(conn = s2n_connection_new(S2N_SERVER));
```

Comment:
Beyond a nit: I've started writing this as
```
struct s2n_connection *conn = s2n_connection_new(S2N_SERVER);
EXPECT_NOT_NULL(conn);
```
because then no person/linter can accidentally accuse "conn" of being uninitialized just bc it was initially declared unintialized.